manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -183,8 +183,16 @@ spec:
               expirationSeconds: 43200
               audience: istio-ca
       - name: istiod-ca-cert
+        {{- if (.Values.env).ENABLE_CLUSTER_TRUST_BUNDLE_API }}
```

Comment:
nit: the API here is a bit awkward since we enable it by setting an env var on the Ztunnel daemon, but ztunnel doesn't actually read the env var at all.

Given its for experimental enablement its not too bad, we are effectively making up for the fact we don't have a first class "feature flag" concept

Reply:
yeah, I didn't want to add a global helm value for this as it's really not something we want to keep around anyway. honestly I would really appreciate first class feature flags, this stuff is always a bit hacky. As soon as a Pilot experimental flag needs to be known about by e.g. the gateways things become weird. Maybe we can pick up that discussion again